opwen_email_server/utils/email_parser.py

```diff
@@ -97,6 +142,41 @@ def _get_image_type(response: Response, url: str) -> Optional[str]:
     return content_type
 
 
+def _get_max_sizes() -> Tuple:
```

Comment:
What does this function give us? Why not use the config values directly at the call site?